README.md

```diff
@@ -1 +1,35 @@
-# shift-left-quality-sample
\ No newline at end of file
+# Welcome to Shift-left-quality discussion
+
+In this branch, I am committing a simple hello-world example in java.
+
+The intent of this branch is to understand how/what are some of the common reveiw cooments that I would be given.
```

Comment:
reveiw = review ( check spelling ) 
cooments = comments ( check spelling ) 
I would be given  == > Check the grammer